index.html

```diff
@@ -0,0 +1,118 @@
+					<li><a href="#">Log out</a></li>
+				</ul>
+				<div class="bar flex-row">
+					<div class="logo flex-row">
+						<img src="./img/Microverse.jpg" alt="">
+					</div>
+					<ul class="bar-mobile-list">
+							<img class="X" src="/img/home/XIcon.svg" alt="close X"></li>
+							<li><a href="about.html">About</a></li>
+						<li><a href="#">Program</a></li>
+						<li><a href="#">Join</a></li>
+						<li><a href="#">Sponsor</a></li>
+						<li><a href="#" id="apply" >APPLY Now</a></li>
+					</ul>
+				<img src="/img/home/hamburgermenu.svg" alt="burgermenu" class="burger">
+			</nav>
+		</header>
+		<main class="card flex-column">
+			<div>
+				<h2 class="card-sub">"Hello World"</h2>
+        <h1 class="card-heading"> BECOME AN INTERNATIONAL DEVELOPER, FROM ANYWHERE, APPLY NOW 2022.</h1>
+			</div>
+			<div class="card-body">
+				<p class="card-text">
+					Become a successful software developer. You can apply for Microverse from any country in the world.
+          Take the first step towards your global career today.
+				</p>
+			</div>
+			<div class="card-footer">
+				<p class="date">2022.12.01(THU) ~ 16(FRI)</p>
+				<p class="venue">&copy; Microverse, Get ready to learn everything</p>
+			</div>
+		</main>
+		<section class="main-program flex-column">
+			<h4 class="program-heading">Main<span> Pro</span>gram</h4>
+			<article class="main-program-items">
+				<div class="program-item">
+					<object data="./img/languages/html.svg"></object>
+					<h5>HTML</h5>
+					<p>
+						The HyperText Markup Language or HTML is the standard markup language for documents designed to be displayed in a web browser.
+					</p>
+				</div>
+				<div class="program-item">
+					<object data="img/languages/css.svg"></object>
+					<h5>CSS</h5>
+					<p>
+						Cascading Style Sheets or CSS is a style sheet language used for describing the presentation of a document written in a markup language.
+					</p>
+				</div>
+				<div class="program-item">
+					<object data="img/languages/javascript.svg"></object>
+					<h5>JavaScript</h5>
+					<p>
+						JavaScript (JS) is a language that enables you to create dynamically updating content, control multimedia.
+					</p>
+				</div>
+				<div class="program-item">
+					<object data="img/languages/ruby.svg"></object>
+					<h5>Ruby</h5>
+					<p>
+						Ruby is an interpreted, high-level, general-purpose programming
+                language which supports multiple programming.
+					</p>
+				</div>
+				<div class="program-item">
+					<object data="/img/languages/github.svg"></object>
+					<h5>GitHub</h5>
+					<p>
+						GitHub, is an Internet hosting service for software development and version control using Git.
+					</p>
+				</div>
+			</article>
+			<footer class="main-footer flex-column">
+				<a href="" class="main-footer-desktop">SEE THE WHOLE PROGRAM</a>
+				<button class="main-footer-mobile">
+					<p href="#">Join<br>&copy; Microverse<br>2022</p>
+				</button>
+			</footer>
```

Comment:
- [ ] Please remove this `footer` tag and replace it with something else. You can only have one footer and it should appear at the bottom of the page which you already have